internal/schema/ize-spec.json

```diff
@@ -145,6 +145,17 @@
             "description": "Serverless apps configuration.",
             "additionalProperties": false
         },
+        "alias": {
+            "id": "#/properties/alias",
+            "type": "object",
+            "patternProperties": {
+                "^[a-zA-Z0-9._-]+$": {
+                    "$ref": "#/definitions/alias"
+                }
+            },
+            "description": "Alias configuration.",
```

Comment:
```suggestion
            "description": "(optional) Alias mode can be enabled here. This can be used to combine various apps via depends_on parameter.",
```